### PR TITLE
Add some usage examples for Bash & Zsh

### PR DESCRIPTION
**Pull request by DBOTW**
... think it would be nice if there are some "ready to run" examples in the repository ...

### CODE REVIEW

**Comment by calleerlandsson** [comment]
Nice! This could also be a great way to resolve #145. Do you have any thoughts on this, @mptre?

**Comment by mptre** [comment]
Great initiative. Is anyone aware of another project doing something
similar? I would prefer a separate file for each subject:

```
$ cat examples/shell.md
# Shell

## History

Pick a command to execute from the history:

```sh
# bash
# XXX Implementation goes here

# zsh
# XXX Implementation goes here
```
```

File: README.md [hunk 1]
```diff
@@ -101,10 +101,12 @@ call `pick(1)` from within Vim, see [the pick.vim source code].
 ***Please note:*** pick requires a fully functional terminal to run and
 therefore cannot be run from within gvim or MacVim.
 
-See the `pick(1)` man page for detailed usage instructions and more examples.
+See the `pick(1)` man page for detailed usage instructions. More examples can be
```
Comment (mptre) [inline README.md]: New sentence -> new line.

File: README.md [hunk 2]
```diff
@@ -101,10 +101,12 @@ call `pick(1)` from within Vim, see [the pick.vim source code].
...
+found in the [examples][examples] directory.
 
 [the pick.vim Vim plugin]: https://github.com/calleerlandsson/pick.vim/
 [the pick.vim source code]: https://github.com/calleerlandsson/pick.vim/blob/master/plugin/pick.vim
+[examples]: examples/
```
Comment (mptre) [inline README.md]: Use an absolute URL.

**Comment by DBOTW** [comment]
> I would prefer a separate file for each subject

The idea was to just `source` a single example file in the `.bashrc` or `.zshrc`:
```sh
source pick-history.zsh
```
and you're done ... ;-)

**Comment by DBOTW** [comment]
> Is anyone aware of another project doing something similar?

https://github.com/junegunn/fzf/tree/master/shell ... but for me this is too overloaded and cryptic ... ;-)

**Comment by mptre** [comment]
Ah ok. I don't want to distribute these files in future releases since
they have to end up at a sensible location on all platforms; putting
more work on our maintainers. If you want to maintain the files by
yourself, I'd be happy to provide a link in the README.

**Comment by DBOTW** [comment]
> If you want to maintain the files by  yourself, I'd be happy to provide a link in the README

Seems that there is no other way ... to be honest, I don't understand your concern as some maintainers include and install the `LICENSE`-file ...

**Pull request closed without merging**